src/math/common.lisp

```diff
@@ -34,3 +34,9 @@
   `(if (< ,n 0)
        "-"
        "+"))
+
+(defmacro append-to (l &rest v)
+  `(setf ,l (append ,l (list ,@v))))
+
+(defmacro macro-curry (func val &key (right nil))
```

Comment:
Ehhh... It was meant to be an example of how you can curry with macro, not an actual useful code.

Reply:
Oh